lgc/CMakeLists.txt

```diff
@@ -62,6 +64,20 @@ option(LLPC_ENABLE_WERROR "Build LLPC with more errors" OFF)
 include(../cmake/CompilerFlags.cmake)
 set_compiler_options(LLVMlgc ${LLPC_ENABLE_WERROR})
 
+### TableGen for LGC dialect ###########################################################################################
+
+include_directories(../imported/llvm-dialects/include)
```

Comment:
Is there a better way of doing this? I tried using target_include_directories on LgcDialectTableGen, but it complained that it is a non-compilable target type.

Reply:
It seems like one can pass `EXTRA_INCLUDES` to `tablegen`? I.e.
```cmake
tablegen(LGC interface/lgc/LgcDialect.h.inc -gen-dialect-decls --dialect lgc
    EXTRA_INCLUDES ../imported/llvm-dialects/include)
```

Reply:
Thanks; just found that myself too.

Reply:
I had to prepend `${CMAKE_CURRENT_SOURCE_DIR}/`. I guess a relative path is relative to whatever the cwd is when it runs tablegen.